services/src/main/java/org/jboss/windup/web/services/service/AnalysisContextService.java

```diff
@@ -129,7 +129,10 @@ public AnalysisContext update(Long analysisContextId, AnalysisContext analysisCo
         this.loadPackagesToAnalysisContext(analysisContext);
         analysisContext.setMigrationProject(original.getMigrationProject());
 
-        return entityManager.merge(analysisContext);
+        analysisContext.getMigrationProject().setProvisional(Boolean.FALSE);
```

Comment:
Added
        // This promotes the provisional project (temporary instance before finished in wizard)
        // to a normal project that is listed in project list.